doc/sphinx/block.rst

```diff
@@ -0,0 +1,22 @@
+Block evaluation functions
```

Comment:
don't' know if this the best name. Can we change block from Block to Vectire/Parallel/Threaded whatever is most appropriate? Block doesn't catch my eye

Reply:
Ehh, I like block...